### DISCUSSION TITLE
Cant delete files

### DISCUSSION THREAD

**Discussion by blablazzz** [aaronwmorris/indi-allsky#1558] [General]
I want to delete files for certain days/nights. i am doing it by going Tools>Generate. Then select action 'Delete images for day' and selecting appropriate date. Check Confirm box and hit Execute. I get message that files are being deleted, but i still see same amount of pictures in the list, i dont see difference in filesystem menu, i still am able to open pics of that day/night. It seems its not deleting. Am i doing smth wrong?

**Comment by aaronwmorris** [comment]
Seems to be broken on my system too.  I will investigate.

**Comment by aaronwmorris** [comment]
Merged #1563 to fix deleting files for date.

**Reply by blablazzz** [reply]
how to apply the fix on my install?

**Reply by aaronwmorris** [reply]
https://github.com/aaronwmorris/indi-allsky/wiki/Updating-indi-allsky
